src/hostapi/wasapi/pa_win_wasapi.c

```diff
@@ -1414,6 +1419,31 @@ static MixMonoToStereoF GetMonoToStereoMixer(const WAVEFORMATEXTENSIBLE *fmtext,
     return NULL;
 }
 
+// ------------------------------------------------------------------------------------------
+static HRESULT TryAllocateMonoMixerBuffer(PaWasapiSubStream *subStream, UINT32 frames)
```

Comment:
I'm not sure what you're getting at with "TryAllocate" here. All allocations can fail. Maybe `ReallocateMonoMixerBuffer` or `ExpandMonoMixerBuffer`

Reply:
`TryXXX` was a leftover from its previous implementation when it was also checking `(subStream->monoMixer != NULL)`, I simplified that to avoid double checks but function name remained the same. Will change to `ReallocateXXX`.